### PR TITLE
Update StorageClassDiagram and ModelClassDiagram

### PR DESCRIPTION
**Pull request by ChenXJ98**

### CODE REVIEW

**Review by joshtyf — COMMENTED** [review]
Looks fine but where are your images?

**Review by joshtyf — APPROVED** [review]
LGTM!

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -34,6 +35,8 @@ MultipleChoiceQuestion *--> "1..*" Choice
 Flashcard *--> "1" Question
 Flashcard *--> "1" Answer
```
Comment (josiahkhoo) [inline docs/diagrams/ModelClassDiagram.puml:35]: It should be OpenEndedQuestion and MultipleChoiceQuestion containing Answer.

**Review by josiahkhoo — APPROVED** [review]
Looks good, just need to make a minor adjustment in the class diagram before we merge this in.

File: docs/diagrams/ModelClassDiagram.puml (hunk 1 above)
Comment (ChenXJ98) [reply docs/diagrams/ModelClassDiagram.puml:35]: yup! Thanks I have updated the diagram!

**Pull request merged**